data/liveinst/liveinst

```diff
@@ -138,13 +140,35 @@ if [ -n "$UPDATES" ]; then
 
 fi
 
+start_anaconda_web_ui() {
+    # remove flag file about initialized backend it shouldn't exists
+    # if the file existed before it might be a broken cleanup
+    if [ -e "$BACKEND_READY_FLAG" ]; then
+        echo "Anaconda backend flag file still exists! Probably a problem with cleanup. Removing now!" >&2
+        rm -f "$BACKEND_READY_FLAG" 2>/dev/null
+    fi
+    # early execution of Firefox to reduce starting time
+    mkdir -p /run/anaconda
+    /usr/libexec/webui-desktop -t live /cockpit/@localhost/anaconda-webui/index.html &
+    echo "$!" > /run/anaconda/webui_script.pid
+}
+
+start_anaconda() {
+    # start Firefox to use Anaconda Web UI if it is installed
+    if [ -e "/usr/share/cockpit/anaconda-webui" ]; then
+        start_anaconda_web_ui
+    fi
+    # start Anaconda main process
+    $ANACONDA "$@"
+}
+
 # Force the X11 backend since sudo and wayland do not mix
 export GDK_BACKEND=x11
 
 if [ -x /usr/bin/udisks ]; then
-    /usr/bin/udisks --inhibit -- "$ANACONDA" "$@"
+    /usr/bin/udisks --inhibit -- start_anaconda "$@"
```

Comment:
Is it necessary to do the additional set up while inhibiting some of the udisks operations? I am not sure.

Reply:
Based on the testing it should be fine. Also we inhibit the udisks for the whole Anaconda run if I'm reading this correctly which included start of the Firefox even before.

@vojtechtrefny do you have some recommendation for this?

Reply:
This must be a very ancient check -- `/usr/bin/udisks` doesn't exists in udisks2, the old udisks package is still available in Fedora, but nobody is using it anymore (udisks 2.0 has been released 11 years ago). I think this can be safely removed.

Reply:
Well the surrounding check evaluates to FALSE anyway and was for some time already. So this can be safely removed.

If you still want to keep the inhibit, you need to to that via D-Bus call to UDisks (2). Unless you want to suspend the system in the middle of installation, I wouldn't worry about that much.

Reply:
heh, ok. In that case let's remove it in separate PR because it doesn't seem related to this. Thanks for info.

And I don't think we need that inhibit, it seems it works fine now so we probably don't need that...